types/cesium/index.d.ts

```diff
@@ -2700,7 +2702,7 @@ declare namespace Cesium {
 
     class LabelGraphics {
         definitionChanged: Event;
-        text: Property;
+        text: ConstantProperty;
```

Comment:
The documentation does not specify that this must be a constant property: https://cesium.com/docs/cesiumjs-ref-doc/LabelGraphics.html?classFilter=label